src/chainparams.cpp

```diff
@@ -370,13 +370,15 @@ bool SelectParamsFromCommandLine()
 // Block height must be >0 and <=last CF reward block height (note that after hfCommunityFundHeight hard fork CF reward is permanent)
 // Index variable i ranges from 0 - (vCommunityFundAddress.size()-1)
 std::string CChainParams::GetCommunityFundAddressAtHeight(int nHeight , Fork::CommunityFundType cfType) const {
-
+    assert(nHeight > 0);
+    assert(nHeight<=consensus.GetLastCommunityRewardBlockHeight());
```

Comment:
This assert should be removed, since height upper limit is a param of getCommunityFundAddress() that is actually used only by fork1 overriding (and an assert is already there)